bedrock/firefox/urls.py

```diff
@@ -30,15 +30,15 @@
     url(r'^firefox/(?:%s/)?(?:%s/)?all/$' % (platform_re, channel_re),
         views.all_downloads, name='firefox.all'),
     page('firefox/accounts', 'firefox/accounts.html'),
-    page('firefox/channel/desktop', 'firefox/channel/desktop.html'),
+    url(r'^firefox/channel/desktop', views.channel_desktop, name='firefox.channel.desktop'),
     page('firefox/channel/android', 'firefox/channel/android.html'),
     page('firefox/channel/ios', 'firefox/channel/ios.html'),
     page('firefox/desktop', 'firefox/desktop/index.html'),
     page('firefox/desktop/fast', 'firefox/desktop/fast.html'),
     page('firefox/desktop/customize', 'firefox/desktop/customize.html'),
     page('firefox/desktop/tips', 'firefox/desktop/tips.html'),
     page('firefox/desktop/trust', 'firefox/desktop/trust.html'),
-    page('firefox/developer', 'firefox/developer.html'),
+    url(r'^firefox/developer', views.developer, name='firefox.developer'),
```

Comment:
@pmac Looks like this URL has broken the "/firefox/developers/notes" redirect for release notes, but I'm not sure what the correct regex should be here.